spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/AxonServerAutoConfiguration.java

```diff
@@ -129,6 +133,13 @@ public CommandPriorityCalculator commandPriorityCalculator() {
         return CommandPriorityCalculator.defaultCommandPriorityCalculator();
     }
 
+    @Bean
+    @ConditionalOnMissingBean
+    public CommandLoadFactorProvider commandLoadFactorProvider(
+            @Value("${axon.axonserver.command.load-factor:100}") int loadFactor) {
```

Comment:
It is better to do this through properties. In that case, IDEs can use autocompletion and properties are automatically documented.

Reply:
Done